admin/class-metabox.php

```diff
@@ -648,6 +648,21 @@ function do_meta_box( $meta_field_def, $key = '' ) {
 			$placeholder = $meta_field_def['placeholder'];
 		}
 
+		// When choosen key is set add chosen class and determine which placeholder should shown.
+		if ( ! empty( $meta_field_def['chosen'] ) ) {
+			$class .= ' yoast-chosen';
+
+			if ( ! empty( $placeholder ) ) {
+				if ( array_key_exists( $placeholder, $meta_field_def['options'] ) ) {
+					$key_to_remove = $placeholder;
+					$placeholder   = $meta_field_def['options'][ $placeholder ];
+					unset( $meta_field_def['options'][ $key_to_remove ] );
+				}
+
+				$placeholder = ' data-placeholder="' . $placeholder . '"';
```

Comment:
Same as previous comment, although I'm suspecting there's a reason why there string is starting with a space.